<launch>
  <arg name="elevation_map_directory" default="$(find-pkg-share autoware_elevation_map_loader)/data/elevation_maps"/>
  <arg name="param_file_path" default="$(find-pkg-share autoware_elevation_map_loader)/config/elevation_map_parameters.yaml"/>

  <node pkg="autoware_elevation_map_loader" exec="elevation_map_loader_node" name="elevation_map_loader" output="screen">
    <remap from="output/elevation_map" to="/map/elevation_map"/>
    <remap from="input/pointcloud_map" to="/map/pointcloud_map"/>
    <remap from="input/pointcloud_map_metadata" to="/map/pointcloud_map_metadata"/>
    <remap from="input/vector_map" to="/map/vector_map"/>
    <remap from="service/get_selected_pcd_map" to="/map/get_selected_pointcloud_map"/>
    <param from="$(var param_file_path)" allow_substs="true"/>
  </node>
</launch>
